etc/packetbeat.yml

```diff
@@ -36,6 +36,26 @@ interfaces:
 
 ############################# Protocols ######################################
 protocols:
+  dns:
+    # Configure the ports where to listen for DNS traffic. You can disable
+    # the DNS protocol by commenting out the list of ports.
+    ports: [53]
```

Comment:
As currently implemented the BPF that is generated is strictly based on the protocols that are implemented by the plugin. For DNS this means that the BPF will be `udp port 53`. When TCP is implemented the BPF filter will generate as `port 53` meaning register for both TCP and UDP. Other modules that only implement TCP generate as `tcp port 80 and tcp port 8080` for example.

There is no way for a user to configure that a plugin should only receive packets from a specific transport. If someone needed to do this they would have to override the BPF.